src/core/parser/csv_parser/csv_parser.cpp

```diff
@@ -110,12 +129,42 @@ std::string CSVParser::GetUnparsedLine(unsigned long long const line_index) {
 }
 
 std::vector<std::string> CSVParser::ParseString(std::string const& s) const {
+    std::size_t const length = s.size();
+    std::string t;
+    for (std::size_t index = 0; index < length; ++index) {
+        if (s[index] == '\\') {  // preserve \ through boost parsing
+            t.append(2, '\\');
+        } else if (s[index] == '"') {  // preserve " through boost parsing
+            t.append("\\\"\"");
```

Comment:
And now quotes are useless, and we may as well use `boost::char_separator` instead of `boost::escaped_list_separator`.
`a,"a,a","a"` should be `a` `"a,a"` `a` but will be parsed as `a` `"a` `a"` `a`
I don't know. Maybe this is ok for now, but using a separate library for CSV parsing seems like the simplest option.